src/iv/ivgl.cpp

```diff
@@ -690,7 +690,9 @@ IvGL::shadowed_text(float x, float y, float /*z*/, const std::string& s,
      * Paint on intermediate QImage, AA text on QOpenGLWidget based
      * QPaintDevice requires MSAA
      */
-    QImage t(size(), QImage::Format_ARGB32_Premultiplied);
+    qreal dpr = devicePixelRatio();
+    QImage t(size() * dpr, QImage::Format_ARGB32_Premultiplied);
+    t.setDevicePixelRatio(dpr);
```

Comment:
hm. I can see in a different place there is 
```
float dpr = m_viewer.devicePixelRatio();
glScalef(dpr, dpr, 1.0f);
```
I need to understand better what all of this is doing ...